### DISCUSSION TITLE
Group anchors

### DISCUSSION THREAD

**Discussion by Robotechnic** [cetz-package/cetz#634] [Q&A] [answered]
I have a question about group anchors: How can I use the (name:...,anchor:...) notation in the anchor parameter of a group?
If I do that:
```typ
#cetz.canvas({
  import cetz.draw: *
  group(
  anchor: (name: "test", anchor: 20deg),
  {
    circle(name: "test", (0,0))
  })
})
```
I'm getting an error:
`Unknown anchor '(name: \"test\")' for element 'none'`
But if I do something like this:
```typ
#cetz.canvas({
  import cetz.draw: *
  group(
  anchor: "test2",
  {
    circle(name: "test", (0,0))
    circle(name: "test2", (name: "test", anchor: 20deg), radius: .1, fill: red)
  })
})
```
It works as expected. 
Is it possible to do such a thing with the first solution?
My problem is that it's hard to use the second solution in what I'm trying to do (it's possible tho, so if the first solution is not doable I can technically use the second one).

**Comment by johannes-wolf** [comment, marked as answer]
You can use the string syntax `"test.20deg"` or create an intermediate anchor insides the group and use that `anchor("o", (name: "test", anchor: 20deg))`.

**Reply by Robotechnic** [reply]
Thanks
